source/helper.h

```diff
@@ -66,6 +75,15 @@ struct Parameters {
   double surface_gravity;
   /* Number of grid points */
   size_t N;
+  
+  /* Arrays for variable abundance radiative losses */
+  double log10_loss_rate_array[101][7][7];
+  double log10_temperature_array[101];
+  double initial_density;
+  double previous_density;
+  double initial_abundance_factor;
+  double previous_abundance_factor;
+  bool upflowing, initial_radiation;
```

Comment:
Could you add brief comments explaining what each of these is? Some are self-explanatory but others (e.g. `bool upflowing`) are not